packages/web-components/src/components/link-list/link-list.ts

```diff
@@ -87,6 +87,10 @@ class DDSLinkList extends StableSelectorMixin(LitElement) {
     window.removeEventListener('resize', this._handleResize);
   }
 
+  updated() {
+    sameHeight(this.childItems, 'md');
+  }
+
```

Comment:
Let's leave this not running for now - We are trying to determine what to do with `sameWidth()`/`sameHeight()` as a whole.

```suggestion
```